### PR TITLE
data: realdezzy

### PR DESCRIPTION
**Pull request by realdezzy**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/realdezzy.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Ikeogu Franklin",
+  "bio": "Blockchain Developer | Python | Solidity | Javascript | Open-source Enthusiast",
+  "avatar": "https://github.com/realdezzy.png",
```
Comment (sandramsc) [inline data/realdezzy.json:4]: The avatar field is no longerrequired
```suggestion

```

**Review by sandramsc — CHANGES_REQUESTED** [review]
Hi, I've added an inline comment with a suggested change.

**Review by SaraJaoude — APPROVED** [review]
I've committed the change regarding the avatar, so I can now merge this PR.

**Comment by realdezzy** [comment]
Thanks for the correction @SaraJaoude I was following an old tutorial

**Pull request merged**